corehq/apps/export/templates/export/customize_export_new.html

```diff
@@ -104,6 +108,9 @@
                             {% trans "Automatically convert dates and links for Excel" %}
                         </label>
                     </div>
+
+                    {% if not use_new_exports %}
```

Comment:
i think this template only gets used if it's use_new_exports. otherwise the `customize_export_old` gets loaded